src/libs/actions/Receipt.js

```diff
@@ -0,0 +1,21 @@
+import Onyx from 'react-native-onyx';
+import ONYXKEYS from '../../ONYXKEYS';
+
+function onUploadReceiptError(isAttachmentInvalid, attachmentInvalidReasonTitle, attachmentInvalidReason) {
```

Comment:
The function name can be improved. Maybe `setUploadReceiptError` would be better.